src/components/sections/SubSectionButtons.vue

```diff
@@ -0,0 +1,40 @@
+<script setup>
```

Comment:
```suggestion
<script setup lang="ts">
```

Pas mal si on normalise tous les composants en TS je pense.